### DISCUSSION TITLE
How is the Netspace calculated?

### DISCUSSION THREAD

**Discussion by FSys77** [Chia-Network/chia-blockchain#5832] [General]
The value that is shown with "chia show -s" (at the time of this posting: `Estimated network space: 8.820 EiB`), how is this calculated?

Is this the sum of all plotfiles that can **theoretically** be farmed? Or is this the number of plotfiles that are **actually** farming? (because of the massive sync issues both numbers should differ quite substantially). Or is it something completely else ...?

**Comment by room101-dev** [comment]
There are lots of chia-explorers out there that integrate all the found new blocks on earth ever 5-15 minutes, they also report the number of plots submitted; So to answer your question the data comes from the chia-explorers, where they average all the plots submitted.

The majority of all plots are on HPOOL in China, +60% last time I looked then another 30% come from china as anonymous, but I think they're  also hpool, then the remaining 10% come from  the west.

Chia-NET (dev) doesn't have a clue how many plots are being' 'farmed' because NOBODY in china uses the Chia-SW for anything, HPOOL provides their own miner, and their own plotter, nobody gives a hell about the chia blockchain, probably one of the big reason it doesn't work is that the entire network is about 'mining' XCH, and not about transactions, or blockchain,

The entire idea of mining has been thrown out  the window, its not about transactions, its all about the plot lottery.

Chia-Net can only estimate how many of its sucker (users) have submitted plots through its harvesting nodes and integrated value, given that is 10% of all plots submitted, that's why the real data is picked up from the explorers, there are also global pool websites that monitor all XCH found, and the submitted plots; all this data is integrated. IMHO CHIA-NET get their EB data from the chia-calc guy, who sells plots, this scam is one big happy family of scammers.
